components/compliance-service/inspec/cli.go

```diff
@@ -30,11 +30,12 @@ const shimBinName = "inspec_runner"
 // variables via debug logs. Leave to false for release.
 const logSensitiveData = false
 
-func inspecShimEnv() map[string]string {
+func inspecShimEnv() (map[string]string, error) {
+	err := AcceptInspecLicense(TmpDir)
```

Comment:
I think it'd make more sense to leave this function as it was and call `AcceptInspecLicense` explicitly where its needed.

Reply:
Talked with Victoria about that yesterday. Thing is `AcceptInspecLicense` is needed everywhere where `inspecShimEnv` is used. I'm going to test a new approach that @tyler-ball mentioned yesterday ( setting env `CHEF_LICENSE=accept-no-persist` )